packages/client/components/ReflectionGroupVoting.tsx

```diff
@@ -118,7 +118,7 @@ const ReflectionGroupVoting = (props: Props) => {
     <UpvoteColumn>
       <UpvoteRow data-cy='reflection-vote-row'>
         <UpvoteIcon
-          data-cy={`remove-vote`}
+          aria-label={`Remove vote`}
```

Comment:
I changed these from `data-cy` variables to aria labels. This should help folks with screen readers understand what the icons represent.